test/state/state.cpp

```diff
@@ -460,6 +480,88 @@ TransactionReceipt transition(const StateView& state_view, const BlockInfo& bloc
     assert(sender_acc.nonce < Account::NonceMax);  // Required for valid tx.
     ++sender_acc.nonce;                            // Bump sender nonce.
 
+    auto delegation_refund = 0;
+
+    for (const auto& auth : tx.authorization_list)
+    {
+        // 1. Verify the chain id is either 0 or the chain’s current ID.
+        // FIXME: Transaction chain id is not the chain id. Can it be 0?
+        if (auth.chain_id != 0 && auth.chain_id != tx.chain_id)
+            continue;
+
+        // 2. Verify the nonce is less than 2**64 - 1.
+        if (auth.nonce == Account::NonceMax)
+            continue;
+
+        // 3. Verify if the signer has been successfully recovered from the signature.
+        //    authority = ecrecover(...)
+        if (!auth.signer.has_value())
+            continue;
+
+        // s value must be less than or equal to secp256k1n/2, as specified in EIP-2.
+        if (auth.s > SECP256K1N_OVER_2)
+            continue;
+
+        // FIXME: In previous impl: If authority exists, but is empty, it will not be warm.
```

Comment:
Add a test case.

Reply:
There is a kinda-invalid test case for it: the one with "empty but with storage" account. In this case, in previous implementation this account will not be marked as warm. The test don't exploit the warmness aspect so this is undetected.